backend/models/mattress.model.ts

```diff
@@ -0,0 +1,12 @@
+import mongoose, {Schema} from "mongoose";
+
+const mattressSchema = new Schema({
+    name: {type: String},
```

Comment:
If any of these properties need to be here than I would advise using something like:
`name: { type: String, required: true }`

Reply:
Ask the question would something break or become an issue if name or any other property was `undefined`